smartcontractkit/external-adapters-js
# ignore empty string env variables

## Closes 14835

## Description

Ignores empty string environment variables

## Changes

 - Update `getEnv` function in the `utils.ts` file to set the environment variable as `undefined` if it is passed an empty string
 - Add unit tests for the `getEnv` function

## Steps to Test

1. Find an adapter that takes a environment variable and set one of them to an empty string
2. Place debugger in `getEnv` function and verify that it returns the environment variable as `undefined`.

## Quality Assurance
